test/Mono.Linker.Tests.Cases/TypeForwarding/UnusedForwarderWithAssemblyCopyIsKept.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+using Mono.Linker.Tests.Cases.TypeForwarding.Dependencies;
+
+namespace Mono.Linker.Tests.Cases.TypeForwarding
+{
+	[SetupLinkerDefaultAction ("link")]
+	[SetupCompileBefore ("Forwarder.dll", new[] { "Dependencies/ReferenceImplementationLibrary.cs" }, defines: new[] { "INCLUDE_REFERENCE_IMPL" })]
+
+	[SetupCompileAfter ("Implementation.dll", new[] { "Dependencies/ImplementationLibrary.cs" })]
+	[SetupCompileAfter ("Forwarder.dll", new[] { "Dependencies/ForwarderLibrary.cs" }, references: new[] { "Implementation.dll" })]
+
+	[SetupLinkerAction ("copy", "Forwarder")]
+
+	[KeptMemberInAssembly ("Forwarder.dll", typeof (ImplementationLibrary))]
+	[RemovedAssembly ("Implementation.dll")]
```

Comment:
How is this test working? I thought that the copy forwarder would result in Implementation being kept due to the reference from the forwarder

Reply:
As discussed; we'll keep the whole copy assembly as is, but otherwise we will not mark anything that isn't either statically referenced or dynamically accessed.

Reply:
Thanks for clarifying offline @mateoatr. For anyone following along - we mark the scope of forwarders in a copy assembly, but don't necessarily mark the resolved type if nothing references it.

Reply:
Oh you beat me to it :)